package.json

```diff
@@ -230,10 +230,20 @@
   },
   "browserslist": {
     "production": [
-      "defaults"
+      "last 2 Chrome versions",
+      "last 2 Firefox versions",
```

Comment:
With firefox and chrome on 6 week release cycles we might want to make this longer. What do you think about 10 versions back? That would give us 60 weeks which is just over 1 year.

Reply:
yeah we can do that, i'm sure it still has more features than the last 3 major ios versions.

Reply:
updated to last 10 versions